easybuild/easyconfigs/g/GSL/GSL-1.16-intel-2016a.eb

```diff
@@ -0,0 +1,18 @@
+easyblock = 'ConfigureMake'
+
+name = 'GSL'
+version = '1.16'
```

Comment:
I'll stick with 1.16 for this, the ROOT version we're using isn't very new (there's a 6.x too), so it's very likely the changes in the GSL API will break stuff.